src/qtiItem/helper/util.js

```diff
@@ -122,10 +122,10 @@ var util = {
         }
 
         if (_.isArray(collectionNames)) {
-            _.each(collectionNames, function(collectionName) {
+            _.each(collectionNames, function (collectionName) {
```

Comment:
https://lodash.com/docs/4.17.15#forEach
```suggestion
            _.forEach(collectionNames, function (collectionName) {
```